src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml

```diff
@@ -10,7 +10,7 @@
         <link rel="stylesheet" href="~/css/site.css" />
```

Comment:
Aren't you missing stuff here?
        <link rel="stylesheet" href="~/Identity/lib/bootstrap/dist/css/bootstrap-grid.css" />
        <link rel="stylesheet" href="~/Identity/lib/bootstrap/dist/css/bootstrap-reboot.css" />
        <link rel="stylesheet" href="~/Identity/lib/bootstrap/dist/css/bootstrap-theme.css" />